packages/ui/src/common/types.ts

```diff
@@ -3,3 +3,4 @@ import { ReactNode } from 'react';
 export type ChildrenList = ReactNode[];
 export type AnyChildren = ReactNode | ChildrenList;
 export type AnyObject = Record<PropertyKey, any>;
+export type WithTestId<TParams> = { testId?: string } & TParams;
```

Comment:
Are we not using interfaces? I expected an interface of `{ testId?: string; }` so other interfaces can extend from.